src/frontend/Komponenter/Behandling/Formkrav/VedtakSelect.tsx

```diff
@@ -58,12 +67,37 @@ export const VedtakSelect: React.FC<IProps> = ({
                 <option value={PåklagetVedtakstype.UTEN_VEDTAK}>
                     {påklagetVedtakstypeTilTekst[PåklagetVedtakstype.UTEN_VEDTAK]}
                 </option>
+                <option value={PåklagetVedtakstype.INFOTRYGD_TILBAKEKREVING}>
+                    {påklagetVedtakstypeTilTekst[PåklagetVedtakstype.INFOTRYGD_TILBAKEKREVING]}
+                </option>
                 {vedtak.sort(sorterVedtakstidspunktDesc).map((valg, index) => (
                     <option key={index} value={valg.eksternBehandlingId}>
                         {fagsystemVedtakTilVisningstekst(valg)}
                     </option>
                 ))}
             </FamilieSelect>
+            {vurderinger.påklagetVedtak.påklagetVedtakstype ===
+                PåklagetVedtakstype.INFOTRYGD_TILBAKEKREVING && (
+                <DatoWrapper>
+                    <Label htmlFor={'vedtaksdato'}>
+                        Vedtaksdato for tilbakekreving i infotrygd
+                    </Label>
+                    <FamilieDatovelger
+                        label={null}
```

Comment:
Blir det feil å bruke label her i stedet for å definiere en egen label? 👀

Reply:
Familiedatovelger sin label er 16px, mens klage ellers har 18. Det så, mildt sagt, teit ut 😂